network/networks.csv

```diff
@@ -13,6 +13,7 @@ GN,GN,The GNS CGPS network,false
 HA,NZ,Hydrophone Array,false
 HB,NZ,Hawke's Bay regional seismic network,false
 IG,LI,LINZ IGS CGPS Network,false
+IU,IU,Global Seismograph Network,false
```

Comment:
would thsat be relavant to add  the IU/GSN DOIt in the description ? 
https://doi.org/10.7914/SN/IU

Reply:
@salichon adding DOIs to delta is being worked on here: https://github.com/GeoNet/delta/pull/2175

Reply:
Okay so i think this would be ann easy way to set ibn 
since the descritpion of the network  is provided with the stationxml